.github/workflows/canister-tests.yml

```diff
@@ -121,6 +121,34 @@ jobs:
             exit 1
           fi
 
+  test-app-build:
```

Comment:
maybe rename to `test-app-build-upload`?

Reply:
I'd rather not, because it would make it inconsistent with `docker-build`.